src/platform/windows/audio.cpp

```diff
@@ -117,19 +117,24 @@ namespace {
 
   using virtual_sink_waveformats_t = std::vector<WAVEFORMATEXTENSIBLE>;
 
+  // The list of virtual formats are sorted in preference order and the first valid format will be used.
+  // All bits-per-sample options are listed because we try to match this to the default audio device.
```

Comment:
This seems like a good opportunity to document the template better. Could you document this template using a doxygen comment block?

```cpp
/*
 * @brief ...
 * @tparam ...
 */
```

- https://docs.lizardbyte.dev/projects/sunshine/latest/md_third-party_2doxyconfig_2docs_2source__code.html#documentation-blocks
- https://stackoverflow.com/a/13359876